replica/replica.go

```diff
@@ -990,7 +998,9 @@ func (r *Replica) createDisk(name string, userCreated bool, created string) erro
 		}
 
 		r.updateChildDisk(oldHead, newSnapName)
-		r.activeDiskData[len(r.activeDiskData)-1].Name = newSnapName
+		// Not Required as diskData and activeDiskData are pointing to same
+		// disks
+		//r.activeDiskData[len(r.activeDiskData)-1].Name = newSnapName
```

Comment:
activeDiskData keeps the file in order, whereas diskData is a map which keeps the data in random order.

Reply:
This is required since it is used in snapshot deletion.